### ISSUE TITLE
Render projection not orthographic

### ISSUE THREAD

**Issue by laurensvalk** [trevorsandy/lpub3d#608]
### Subject
Even though the visual editor looks correct and orthographic, the render is not orthographic:

![image](https://user-images.githubusercontent.com/12326241/193536034-de43dd33-0a3b-4ee9-9bf9-d596da78792d.png)

![image](https://user-images.githubusercontent.com/12326241/193536612-f99b38bc-31f7-45e9-ae86-9f90cc9649ee.png)

![image](https://user-images.githubusercontent.com/12326241/193536468-b8a8db7b-530f-4c25-bd81-3d0caa4a9e5a.png)

If I switch "viewpoint" to "front" like this:

![image](https://user-images.githubusercontent.com/12326241/193535312-18e0efbb-83bb-4a1a-8352-10482d6f8ca6.png)

Then it looks OK:

![image](https://user-images.githubusercontent.com/12326241/193536190-0ddcb4b6-e80f-49d4-985b-ff9f05acc3fa.png)

Is this expected behavior? What is the difference between "front" and "default"? Thanks!

On some parts, I see no difference in perspective at all.

### Environment
I am running the `LPub3D-2.4.4.0.2671_20220531-x86_64.AppImage` on Ubuntu 22.04 without any changes.
```
0 New Model
0 Name: pupdevice-color.ldr
0 Author: MLCad
0 Unofficial Model
0 ROTATION CENTER 0 0 0 1 "Custom"
0 ROTATION CONFIG 0 0
1 15 0 0 0 -1 0 0 0 -1 0 0 0 1 37308c01.dat
0 ROTSTEP 35 -35 0 ABS
0
```

**Comment by trevorsandy** [comment]
Thank you for reporting this behaviour.

> Is this expected behavior?

No, the viewpoint should be set to Default when using projection rendering (i.e. orthographic). The settings should automatically enable and disable the opposing setting accordingly. If you click help (?) on at the top right corner of the dialog (release v2.4.5 or greater), you will see the following guidance on how these settings are supposed to work.

> What is the difference between "front" and "default"?

Basically, Default is equivalent to 'Do not do this' - so if viewpoint is set to Default, it is indicating to the application not to use viewpoint rendering - use the current projection render setting. Alternatively, when projection is set to Default, it is indicating to not to use this rendering mode - use viewpoint rendering.

I have updated the settings to automate enabling and disabling accordingly.

Cheers,
